### PR TITLE
16 add docker mounted volume paths to constants.py

### PR DESCRIPTION
**Pull request by yaseminbridges**
Added constants for docker target directory to be used and called within the code

### CODE REVIEW

**Comment by matentzn** [comment]
Can you link this to an issue? I don't really understand what exactly this is here; why would any variable be 'docker' specific?

**Comment by yaseminbridges** [comment]
> Can you link this to an issue? I don't really understand what exactly this is here; why would any variable be 'docker' specific?

It is already linked to an issue but I added more of a description. I'm not an expert in the Docker terminology but hopefully it makes more sense

File: src/pheval_exomiser/constants.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PHENOPACKET_TARGET_DIRECTORY_DOCKER = "/exomiser-testdata-phenopacket/"
+RAW_RESULTS_TARGET_DIRECTORY_DOCKER = "/exomiser-results/"
+OUTPUT_OPTIONS_TARGET_DIRECTORY_DOCKER = "/exomiser-testdata-output-options/"
+VCF_TARGET_DIRECTORY_DOCKER = "/exomiser-testdata-vcf/"
+EXOMISER_YAML_TARGET_DIRECTORY_DOCKER = "/exomiser-yaml-template/"
+EXOMISER_DATA_DIRECTORY_TARGET_DOCKER = "/exomiser-data"
```
Comment (julesjacobsen) [inline src/pheval_exomiser/constants.py:6]: missing trailing path separator?

File: src/pheval_exomiser/constants.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+INPUT_COMMANDS_TARGET_DIRECTORY = "/exomiser-batch-file/"
+EXOMISER_CONFIG_TARGET_DIRECTORY = "/exomiser-config/"
```
Comment (julesjacobsen) [inline src/pheval_exomiser/constants.py:8]: Add `_DOCKER` suffix to maintain naming consistency?

**Comment by yaseminbridges** [comment]
implemented changes but failing as now need PR #28 to be merged into branch and those changes merged into here

**Comment by julesjacobsen** [comment]
Did you do a successful manual test run on some real data? If so you need to update the test cases to fix the failing tests.

**Comment by yaseminbridges** [comment]
> Did you do a successful manual test run on some real data? If so you need to update the test cases to fix the failing tests.

I just needed to merge the main branch into here and fix two of the tests, these next GitHub actions should all pass hopefully

**Pull request merged**